lib/deqm_test_kit/patient_everything.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+require 'json'
+
+module DEQMTestKit
+  # Perform Patient/$everything operation on test client
+  class PatientEverything < Inferno::TestGroup
+    id 'patient_everything'
+    title 'Patient/$everything'
+    description 'Ensure fhir server can respond to the Patient/$everything request'
+
+    fhir_client do
+      url :url
+    end
+
+    TEST_PATIENT_ID = 'test-patient'
+
+    test do
+      title 'Patient/<id>/$everything valid submission'
+      id 'patient-everything-01'
+      description 'Patient data is received for single patient on the server'
+      makes_request :patient_everything
```

Comment:
The use of makes_request in this suite isn't quite correct. makes_request is used to save requests that you may want to reuse later. Since I don't think we are reusing any of the requests in this suite you can get rid of all the makes_requests and the names in the fhir operations for each of these tests